fuzzing_tc/common/pool.py

```diff
@@ -64,6 +64,61 @@
 ARCHITECTURES = frozenset(("x64", "arm64"))
 
 
+def parse_size(size):
+    """Parse a human readable size like "4g" into (4 * 1024 * 1024 * 1024)
+
+    Args:
+        size (str): size as a string, with si prefixes allowed
+
+    Returns:
+        float: size with si prefix expanded
+    """
+    match = re.match(r"\s*(\d+\.\d*|\.\d+|\d+)\s*([kmgt]?)b?\s*", size, re.IGNORECASE)
+    assert match is not None, "size should be a number followed by optional si prefix"
+    result = float(match.group(1))
+    multiplier = {
+        "": 1,
+        "k": 1024,
+        "m": 1024 * 1024,
+        "g": 1024 * 1024 * 1024,
+        "t": 1024 * 1024 * 1024 * 1024,
+    }[match.group(2).lower()]
+    return result * multiplier
+
+
+def parse_time(time):
```

Comment:
Using time as a name is a bit of a footgun.

Reply:
I don't see this function changing enough to make this dangerous. The `time` module isn't used in this entire project because we need to work with timezone aware `datetime` objects. Even if we did: flake8 would catch it.